x-pack/test_serverless/functional/test_suites/common/home_page.ts

```diff
@@ -11,9 +11,11 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
   const svlCommonPage = getPageObject('svlCommonPage');
   const svlCommonNavigation = getService('svlCommonNavigation');
 
-  // Failing: See https://github.com/elastic/kibana/issues/165386
-  // FLAKY: https://github.com/elastic/kibana/issues/165414
-  describe.skip('home page', function () {
+  describe('home page', function () {
+    before(async () => {
+      await svlCommonPage.login();
```

Comment:
Is there a reason, this suite doesn't logout in an `after` block?

Reply:
Good catch, sending update!